src/Process/3dTilesProcessing.js

```diff
@@ -120,21 +117,13 @@ function computeNodeSSE(camera, node) {
 }
 
 export function init3dTilesLayer(context, layer) {
-    if (layer.initialised) {
-        return;
-    }
-    layer.initialised = true;
-
-    Fetcher.json(layer.url, layer.networkOptions).then((tileset) => {
-        const urlPrefix = layer.url.slice(0, layer.url.lastIndexOf('/') + 1);
-        layer.tileIndex = new $3dTilesIndex(tileset, urlPrefix);
-        layer.asset = tileset.asset;
-        requestNewTile(context.view, context.scheduler, layer, tileset.root, undefined).then(
-            (tile) => {
-                layer.object3d.add(tile);
-                tile.updateMatrixWorld();
-                layer.root = tile;
-            });
+    layer.preprocessedPromise.then((layer) => {
```

Comment:
Actually maybe we should keep `init3dTilesLayer`, but replace its usage in `preUpdate` with something like this:
(done in `3dtiles.html` ?)
```js
layer.preprocessedPromise.then((layer) => {
   itowns.$3dtiles.init3dTilesLayer(layer);
}
```